scripts/e2e-ci.sh

```diff
@@ -109,12 +117,12 @@ test_init() {
 
         step Setting small-size SDG-specific config
         python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" generate.pipeline simple
-        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" generate.teacher.model_path "${CACHE_HOME}/instructlab/models/${MERLINITE_GGUF_MODEL}"
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" generate.teacher.model_path "${MODEL_CACHE}/${HUGGINGFACE_DIR}/${MERLINITE_GGUF_REPO}/main"
```

Comment:
I'd prefer you either keep using `CACHE_HOME` or make `MODEL_CACHE` part of the `init_e2e_tests` function - same comment applies to all `test_init` changes